autopilot/ap-batch-transform.ipynb

```diff
@@ -0,0 +1,1297 @@
+{
```

Comment:
<p>Download the file from s3://sagemaker-example-files-prod-REGION/datasets/tabular/uci_bank_marketing/bank-additional-full.csv</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>